nginx/setup.sh

```diff
@@ -1,20 +1,65 @@
+#!/bin/bash
+
+# Setup Nginx for local developement
+#
+# Copies Nginx configuration and SSL certificates to Nginx home (usually /usr/local/etc/nginx)
+# Make sure you have valid AWS credentials and then run with sudo setup.sh <profile name>
+# Remember to add nginx/hosts to your /etc/hosts
+
+S3_BUCKET="s3://identity-local-ssl/"
```

Comment:
This is indeed a pre-existing bucket in Identity, so I just re-used it. (Note that other projects like DAPI would need access to this cert.)